lib/apiservers/service/swagger.json

```diff
@@ -198,14 +198,15 @@
       "get": {
         "summary": "Access log messages for a VCH",
         "description": "Making a `GET` request on /log under a VCH resource will return the log messages for vic-machine processes on a VCH",
+        "produces": [ "text/plain" ],
         "parameters": [
           { "$ref": "#/parameters/target" },
           { "$ref": "#/parameters/vch-id" },
           { "$ref": "#/parameters/thumbprint" }
         ],
         "responses": {
           "200": { "$ref": "#/responses/vch-log" },
-          "default": { "$ref": "#/responses/error" }
+          "default": { "description":"Error when getting VCH logs", "schema": {"type": "string" } }
```

Comment:
I think it might make sense to define a response for this (e.g., `#/responses/error-plain`) to avoid duplication between the two handlers.

Reply:
Thanks, just added the change